pyrealm/pmodel/pmodel.py

```diff
@@ -504,8 +343,20 @@ def gs(self) -> NDArray:
         self._check_estimated("gs")
         return self._gs
 
+    @property
+    def ppfd(self) -> NDArray:
+        """Stomatal conductance (µmol m-2 s-1)."""
```

Comment:
I'm not sure how propeties work, but should the docstring not say "PPFD" here, and "fARAR" in the next one?

Reply:
It should!